examples/Dockerfile.hip-clang

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get clean && apt-get update && apt-get install -y locales
+RUN locale-gen en_US.UTF-8
+RUN update-locale LANG=en_US.UTF-8
+ENV LC_ALL C.UTF-8
+ENV LANG C.UTF-8
+
+# Install rocm
+RUN apt-get update && apt-get install -y gnupg2 --no-install-recommends curl && \
+  curl -sL http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | apt-key add - && \
+  sh -c 'echo deb [arch=amd64] http://repo.radeon.com/rocm/apt/3.7/ xenial main > /etc/apt/sources.list.d/rocm.list'
+
+RUN apt-get update &&\
+    apt-get install -y sudo git bash build-essential cmake rocm-dkms libpython3.6-dev python3-pip miopen-hip rocblas half 
+
+# Install rbuild
+RUN pip3 install https://github.com/RadeonOpenCompute/rbuild/archive/master.tar.gz
+
+ENV PATH /opt/miniconda/bin:/code/cmake-3.14.3-Linux-x86_64/bin:${PATH}
+
```

Comment:
Looks like we did not  install the cmake-3.14.3 in this dockerfile, maybe you can add a line like:

RUN mkdir /code && cd /code && \
    wget https://github.com/Kitware/CMake/releases/download/v3.14.3/cmake-3.14.3-Linux-x86_64.sh && \
    chmod +x cmake-3.14.3-Linux-x86_64.sh && \
    ./cmake-3.14.3-Linux-x86_64.sh --prefix=/usr/local --exclude-subdir --skip-license

You may also need to include the wget in the apt install list

Reply:
I think this was a remnant of taking your onnx runtime dockerfile. Is it worthwhile to use a newer version of cmake in this dockerfile? If so, then I'll make changes to install it.